### PR TITLE
Complete the task but have problem with hover 

### PR DESCRIPTION
**Pull request by barsuchellos**
I dont know what to do with my hover , because in the test my hover have problem but on live server there  problem is no such 

- [DEMO LINK](https://barsuchellos.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://barsuchellos.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+  color: #060b35;
+  font-weight: 500;
+}
+
+.info-code {
+  margin-top: 4px;
+  color: #616070;
+  font-size: 10px;
+  line-height: 14px;
+}
```
Comment (Dikey945) [inline src/style.css:50]: This class naming does not comply with the BEM methodology

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+
+.stars:last-child {
+  margin-right: 0;
+}
+
+.rating_stars > .stars:nth-child(-n+4) {
+  background-image: url(./images/star-active.svg);
+}
+
```
Comment (Dikey945) [inline src/style.css:80]: you should transfer the whole stars block, use modificators to choose right stars raw

File: src/index.html [hunk 3]
```diff
@@ -1,13 +1,42 @@
...
+  <body>
+    <section data-qa="card" class="product-card">
+      <img
+        src="./images/imac.jpeg"
+        alt="imac"
+        class="product-card__logo"
+      >
+      <div class="product-card__info">
+        <p class="info_name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</p>
+        <p class="info-code">Product code: 195434</p>
```
[the remark this answers is not in the document]
Comment (barsuchellos) [reply src/index.html:20]: ok , I will fix it)

File: src/style.css (hunk 1 above)
Comment (barsuchellos) [reply src/style.css:50]: fixing...

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+.product-card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+  height: 406px;
+  transition: 0.3s;
```
[the remark this answers is not in the document]
Comment (MaksimusCRS) [reply src/style.css:16]: What about this comment?

**Review by MaksimusCRS — CHANGES_REQUESTED** [review]

File: src/style.css (hunk 4 above)
[the remark this answers is not in the document]
Comment (barsuchellos) [reply src/style.css:16]: yes , i forgot to write answer, i removed my transition , because in task  didnt write about it

**Review by MaksimusCRS — CHANGES_REQUESTED** [review]
But you need to add transition, it's required

File: src/index.html [hunk 5]
```diff
@@ -1,13 +1,51 @@
...
+        <img
+          src="./images/imac.jpeg"
+          alt="imac"
+          class="product-card__logo product-card__logo--position-margin"
+        >
+        <div class="content">
+          <p class="content__name">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </p>
+          <p class="content__code content__code--position-bottom">Product code: 195434</p>
```
Comment (Denchepornyuk) [inline src/index.html:25]: the same here, you can add margin 16px to `content__code` element, don't create modifier for margins

File: src/index.html [hunk 6]
```diff
@@ -1,13 +1,51 @@
...
+
+          <div class="content__price content__price--position-bottom">
+            <p class="content__text">Price:</p>
+            <p class="content__num">$2,199</p>
+          </div>
+          <a href="#">
+            <button data-qa="hover" class="content__button">
+              Buy
+            </button>
+          </a>
```
Comment (Denchepornyuk) [inline src/index.html:45]: don't write `button` inside `a` use `button` or `a` here

**Review by Denchepornyuk — CHANGES_REQUESTED** [review]

File: src/index.html [hunk 7]
```diff
@@ -1,13 +1,51 @@
...
+      <div data-qa="card" class="product-card">
+        <img
+          src="./images/imac.jpeg"
+          alt="imac"
+          class="product-card__logo product-card__logo--position-margin"
+        >
+        <div class="content">
+          <p class="content__name">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </p>
```
Comment (Denchepornyuk) [inline src/index.html:24]: this should be title, use tags for title

File: src/index.html [hunk 8]
```diff
@@ -1,13 +1,51 @@
...
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <main class="page">
+      <div data-qa="card" class="product-card">
+        <img
+          src="./images/imac.jpeg"
+          alt="imac"
+          class="product-card__logo product-card__logo--position-margin"
```
[the remark this answers is not in the document]
Comment (barsuchellos) [reply src/index.html:19]: ok , i will fixing it

File: src/index.html (hunk 6 above)
Comment (barsuchellos) [reply src/index.html:45]: fixing

File: src/index.html (hunk 7 above)
Comment (barsuchellos) [reply src/index.html:24]: fixing

**Review by Denchepornyuk — APPROVED** [review]

**Pull request closed without merging**